sip/execution_framework_interface/docker_api/client/docker_client.py

```diff
@@ -0,0 +1,470 @@
+# -*- coding: utf-8 -*-
+"""Docker Client API"""
+
+import re
+import os
+import logging
+import docker
+import yaml
+
+
+LOG = logging.getLogger('SIP.EFI.DC')
+
+
+class DockerClient:
+    """Docker Client Interface"""
+
+    def __init__(self):
+        """ Constructor of the class"""
+
+        # Create a docker client
+        self._client = docker.from_env()
+
+        # Store a flag to show whether we are on a manager node or a worker.
+        self._manager = self._client.info()['Swarm']['ControlAvailable']
+
+        # Docker low-level API
+        self._api_client = docker.APIClient()
+
+    ###########################################################################
+    # Create functions
+    ###########################################################################
+
+    def create_services(self, compose_files):
```

Comment:
What is the use case for multiple compose files? Unless there is a good one this function is probably over-engineered.